xlive/H2MOD/GUI/imgui_integration/Console/CommandCollection.cpp

```diff
@@ -769,3 +776,56 @@ void CommandCollection::DeleteObject(datum objectDatumIdx)
 		Engine::Objects::object_destroy(objectDatumIdx);
 	}
 }
+
+int CommandCollection::map_name(const std::vector<std::string>& tokens, ConsoleCommandCtxData cbData)
+{
+	std::string name(tokens[1].begin(), tokens[1].end());
+	main_game_launch(name.c_str());
```

Comment:
Why not just access the token directly, in the argument list, instead of creating another string?

Reply:
Stack corruption happens when you try and do that when optimized

Reply:
How could that cause stack corruption?

Reply:
No clue, something to do with cleanup after the map_name function is called from the console